### ISSUE TITLE
Support for external file structure of binary data for the SIARD 2.0 format

### ISSUE THREAD

**Issue by andersbonielsen** [keeps/db-preservation-toolkit#212]
**Suport for LOBS stored outside the SIARD archive file**
According to appendix D in E-ARK deliverable 3.3 - SIP Pilot specification
http://www.eark-project.com/resources/project-deliverables/51-d33pilotspec
Latest version of spec. (with changes req. by Bruno)
https://docs.google.com/document/d/1FPTm1JIOJkjXX0OFVkPH_T9duAKjv7CD-pAuCTGV0qY/edit#

Parameters needed:
Set storage of LOBs to be inside (SIARD 1.0) or outside (SIARDDK and SIARD 2.0) the siard archive file. (boolean)
Set limit for folder size - before creating new folder (max total size of files per folder in MB) 
Set limit for folder amount - before creating new folder (max no. of files per folder)

Needed for E-ARK pilot 1
Needed by mid July 2016

**Comment by chalkos** [comment]
This is already implemented for SIARD 2. From the usage text for SIARD 2 export module:

```
-eel, --export-external-lobs    (optional) Saves any LOBs outside the siard file.
-eelpf, --export-external-lobs-per-folder=value    (optional) The maximum number of files present in an external LOB folder. Default: 1000 files.
-eelfs, --export-external-lobs-folder-size=value    (optional) Divide LOBs across multiple external folders with (approximately) the specified maximum size (in Megabytes). Default: do not divide.
```

The way it works is passing it the `-eel` parameter to activate the "store LOBs outside" functionality. Then use the defaults (create a new folder every 1000 files, disregarding the size) or specify the "max no. of files per folder" (with `-eelpf value`) and/or the "max total size of file per folder in MB" (with `-eelfs size`).

I have just updated the "Application usage" wiki page with these 3 parameters, which I must have forgotten to add when I finished this feature. They were present on the application usage help text that is displayed when executing the application without arguments or with `-h` (for help).

The title refers only SIARD2, in which this feature is already implemented. But you also refer that SIARD-DK module should also have this feature, which I don't think it does. Should this issue be about implementing this functionality in SIARD-DK?

We can also continue to discuss the feature. Specially if there have been significant changes to the "external LOBs recommendation" since I implemented it for SIARD 2 modules.
